pages/portfolio.html

```diff
@@ -2,11 +2,58 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="../css/protfolioStyle.css"/>
+  <title>Yanxian Zhang</title>
 </head>
 <body>
-  
+  <div class="headMessage">
+    <nav class="menu">
+      <ul>
+        <a href="index.html" target="_top"> Home </a>
+        <a href="about.html" target="_top"> About </a>
+        <a href="portfolio.html" target="_top"> PROTFOLIO </a>
+      </ul>
+    </nav>
+  </div>
+
+  <div class="outer_container">
+    <div class="inner_container">
+
+      <div class="project">
+        <div class="project_pic">
+          <a href="https://github.com/yxzhang88/personal-portfolio-site" target="_blank"> <img src="../assets/view.png" alt="viewing party" height="300px"> <button>Viewing Party</button> </a>
```

Comment:
1. The height attribute of the `img` element assumes it is pixels, so you should not add `px` at the end of the number. It causes an error in the HTML validator.
2. The `button` element cannot be inside an `a` element. The `a` tag already makes it click-able, you do not need to say it is a button. This causes errors.
3. The alt text for images should be more descriptive. People who are blind or have low vision depend on screen readers to tell them what is on the page. Screen readers use the alt text to explain to the user what the picture is about. Just saying "viewing party" does not explain to the user what the picture is about.